editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -114,6 +109,11 @@ HelpBrowser::HelpBrowser(QWidget* parent): QWidget(parent) {
 }
 
 void HelpBrowser::onPageLoad() {
+    // add these actions to weview's renderer, to capture shift+enter and possibly other swallowed shortcuts
+    ((OverridingAction*)mActions[EvaluateRegion])->addToWidget(mWebView->focusProxy());
+    ((OverridingAction*)mActions[Evaluate])->addToWidget(mWebView->focusProxy());
```

Comment:
always prefer the least powerful cast, here it's static_cast:

```suggestion
    static_cast<OverridingAction*>(mActions[EvaluateRegion])->addToWidget(mWebView->focusProxy());
    static_cast<OverridingAction*>(mActions[Evaluate])->addToWidget(mWebView->focusProxy());
```